test/integration/controllers/offer.spec.js

```diff
@@ -116,4 +151,40 @@ describe('Offer controller', () => {
       expectError(404, OFFER_NOT_FOUND, response)
     })
   })
+
+  describe(`shoud return min and mix prices ${endpointUrl}`, () => {
+    it('should throw OFFER_NOT_FOUND', async () => {
+      const response = await app
+        .get(endpointUrl + 'price-values/?authorRole=')
+        .set('Authorization', `Bearer ${accessToken}`)
+
+      expectError(404, OFFER_NOT_FOUND, response)
+    })
+
+    it('should return min and max prices for student offers', async () => {
+      const response = await app
+        .get(endpointUrl + 'price-values/?authorRole=student')
```

Comment:
pretty sure its gonna be price-values?authorRole=student

Reply:
I didn't get the point. Please paraphrase.

Reply:
As far as I understood, the `authorRole` comes from the query, so your URL should look like `url?role=student`, without a slash (not `url/?role=student`).

in other words: remove a slash before ?

Reply:
fixed